packages/textinput/src/react/index.js

```diff
@@ -1,55 +1,57 @@
-import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
-import * as glamor from 'glamor'
-import Halo from '@pluralsight/ps-design-system-halo/react'
+import { css } from 'glamor'
 import PropTypes from 'prop-types'
-import Icon from '@pluralsight/ps-design-system-icon/react'
 import React from 'react'
-import { useTheme } from '@pluralsight/ps-design-system-theme/react'
 
-import css from '../css/index.js'
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+import Halo from '@pluralsight/ps-design-system-halo/react.js'
+import Icon from '@pluralsight/ps-design-system-icon/react.js'
+import { useTheme } from '@pluralsight/ps-design-system-theme/react.js'
+
+import stylesheet from '../css/index.js'
 import * as vars from '../vars/index.js'
 
 const styles = {
-  error: _ => glamor.css(css['.psds-text-input__error']),
+  error: _ => css(stylesheet['.psds-text-input__error']),
```

Comment:
Nice catch!  I'm thinking we shouldn't have crossed these all off our list so quickly.  Seems like there were still some to style update.

Reply:
yeah....i'm planning on doing a final audit/fix. soon hopefully 🤞